app/profile/profileActions.ts

```
@@ -0,0 +1,18 @@
+'use server'
+import prisma from "@/lib/prisma"
+import { userSchema } from "./userSchema"
+import { revalidatePath } from "next/cache"
+
+export async function updateUser(data: any) {
+    debugger
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)